packages/terra-avatar/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ChangeLog
 =========
+### Changed
```

Comment:
oops...this needs moved under the unreleased section

Reply:
updated here https://github.com/cerner/terra-core/pull/2810/commits/ade62bfd9f0af64bf09835d5432378e52191c617.